src/daemon/scheduler.rs

```diff
@@ -172,18 +176,19 @@ impl Scheduler {
                 self.tasks.sync_parent(
                     ca.handle().clone(),
                     parent.clone(),
-                    self.config.ca_refresh_start_up(use_jitter || too_many_parents),
+                    self.config.ca_refresh_start_up(too_many_cas || too_many_parents),
                 );
             }
 
-            // Plan a sync with the repo. In case we only have a handful of CAs
-            // then the result is that the sync is scheduled asap. Otherwise
-            // spread the load.
-            // Note: if circumstances dictate a sync before it's planned, e.g.
-            // because ROAs are changed, then it will be rescheduled accordingly.
-            // Note: users can override using the 'bulk' functions.
-            self.tasks
-                .sync_repo(ca.handle().clone(), self.config.ca_refresh_start_up(use_jitter));
+            // Plan a sync with the repo. But only in case we only have a handful
+            // of CAs.
+            //
+            // Note: if circumstances dictate a sync e.g. because ROAs are changed,
+            // then it will be scheduled accordingly. Furthermore, users can the 'bulk'
+            // function to explicitly force scheduling a sync.
+            if !too_many_cas_resync_repo {
+                self.tasks.sync_repo(ca.handle().clone(), now());
```

Comment:
Do I understand correctly that jitter is no longer used here because it's only now done if there are very few CAs and jitter is only needed when there are lots of CAs?

Reply:
indeed